bcipy/signal/model/offline_analysis.py

```diff
@@ -114,7 +116,10 @@ def offline_analysis(data_folder: str = None,
         offline_analysis_tone = parameters.get('offline_analysis_tone')
         play_sound(offline_analysis_tone)
 
-    return model
+    if return_fig_handles:
+        return model, fig_handles
+    else:      
+        return model
```

Comment:
This creates an awkward API in that the type signature depends on the parameters passed. In the case where we don't want the figure handles maybe this should return `model, None`, so we can still expect to get a tuple.

Reply:
Alternatively it could just always return, and discard where unwanted? Happy to do whichever you prefer

Reply:
That works for me!